kubernetes/linera-validator/build_and_redeploy.sh

```diff
@@ -85,13 +85,23 @@ if [ -n "$cloud_mode" ]; then
 else
     docker_image="linera-test:latest"
     if [ -n "$do_build" ]; then
-        docker build \
-            -f ../../docker/Dockerfile \
-            ${copy+--build-arg binaries=target/release} \
-            --build-arg environment=k8s-local \
-            --build-arg target="$(uname -m)-unknown-linux-gnu" \
-            ../../ \
-            -t "$docker_image"
+        if [ "$(uname -m)" = "x86_64" ]; then
+            docker build \
+                -f ../../docker/Dockerfile \
+                ${copy:+--build-arg binaries=target/release} \
+                --build-arg environment=k8s-local \
+                --build-arg target="x86_64-unknown-linux-gnu" \
+                ../../ \
+                -t "$docker_image" || exit 1
+        else
+            docker build \
+                -f ../../docker/Dockerfile \
+                ${copy:+--build-arg binaries=target/release} \
+                --build-arg environment=k8s-local \
+                --build-arg target="aarch64-unknown-linux-gnu" \
+                ../../ \
+                -t "$docker_image" || exit 1
+        fi
```

Comment:
Great catch on `uname -m`! :grin: 

There's a lot of duplication here.  I might prefer just saying:
```suggestion
        arch="$(uname -m)"
        docker build \
            -f ../../docker/Dockerfile \
            ${copy:+--build-arg binaries=target/release} \
            --build-arg environment=k8s-local \
            --build-arg target="${arch/#arm/aarch}"-unknown-linux-gnu \
            ../../ \
            -t "$docker_image" || exit 1
```
since the arch is the only thing that changes.

Reply:
Ah, nice! More bash syntax I wasn't familiar with 🤣 thank you!